### PR TITLE
[Concept entry] AI Large Language Model

### LINKED ISSUE #2108: [Concept Entry] AI Large Language Models
### Reason/inspiration (optional)

We would like a new topic entry for Large Language Models (LLMs).

The entry should go in a new file in docs/content/ai/concepts/ai-models/models/ai-large-language-models/ai-large-language-modelsmd

The entry should
- Describe what a Large Language Model is.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Hassan-fayez**
Describe what a Large Language Model is.

### Description

-added a new concept to describ the AI LLM
Closes #2108 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/ai /concepts/ai-models/models/ai-large-language-models/ai-large-language-modelsmd [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Large Language Models (LLMs)
+
+A **Large Language Model (LLM)** is an artificial intelligence system that uses natural language processing (NLP) techniques to generate human-like text. LLMs are based on deep learning models that are trained on massive amounts of text data, allowing them to understand and generate complex language patterns.
```
Comment (SSwiniarski) [inline content/ai /concepts/ai-models/models/ai-large-language-models/ai-large-language-modelsmd:3]:
```suggestion
A **Large Language Model (LLM)** is an artificial intelligence system that uses natural language processing (NLP) techniques to generate human-like text. LLMs are based on deep learning models that are trained on massive amounts of text data, allowing them to generate complex language patterns.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Hassan-fayez I did a first review and left some comments. The most important is the missing metadata and updating the file name.

**Comment by Hassan-fayez** [comment]
hi @SSwiniarski :) 

thank you for your explanation, i realized that i pushed the wrong branch to the PR.
i have submitted a new  [PR](https://github.com/Codecademy/docs/pull/2165) with the correct metadata and path

**Comment by SSwiniarski** [comment]
If you're submitting another PR for this, I am closing this one.

**Pull request closed without merging**